activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ExtensionsVariablesMappingProvider.java

```diff
@@ -139,6 +147,47 @@ private Optional<Object> calculateOutPutMappedValue(Mapping mapping,
         return Optional.empty();
     }
 
+    private Optional<Object> patchVariable(Object changesToApply, Object processVariableCurrentValue) {
+        try {
+            JsonNode oldNode;
+            if (isProcessVariableNull(processVariableCurrentValue)) {
+                oldNode = objectMapper.createObjectNode();
+            } else {
+                oldNode = objectMapper.convertValue(processVariableCurrentValue, JsonNode.class);
+            }
+
+            JsonNode patchNode = objectMapper.convertValue(changesToApply, JsonNode.class);
+            ensurePathExists(oldNode, patchNode);
+
+            JsonNode patchedNode = JsonPatch.apply(patchNode, oldNode);
+
+            Object updatedObject = objectMapper.treeToValue(patchedNode, Object.class);
+            return Optional.ofNullable(updatedObject);
+        } catch (Exception e) {
+            throw new ActivitiIllegalArgumentException("Invalid jsonPatch variable mapping", e);
```

Comment:
WDYT about adding inputs to expection or maybe log it?

Reply:
Good point. I will add a log message otherwise this will be hard to debug